tools/local-snapshot-runner-app/src/logger/FileLogger.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as fs from "fs";
+import { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+
+/**
+ * TODO: Look at FileLogger class in stress tests
+ */
+export default class FileLogger implements ITelemetryBaseLogger {
+    public supportsTags?: true | undefined;
+
+    public constructor(
+        private readonly fileName: string,
+    ) { }
+
+    /**
+     * Appending each line to file right away for now
+     * @param event - TODO
+     */
+    public send(event: ITelemetryBaseEvent): void {
+        const logEvent = JSON.stringify(event);
+        console.log(logEvent);
+
+        fs.appendFileSync(this.fileName, `${logEvent}\n`);
```

Comment:
That might be terribly slow - appends on most OSes is not fast. FileLogger class in service tests assumes accumulation and flushing on some cadence / end of test. I like it better, though there is a risk of losing data.
I'd say two things:
- As long as we know how to collect data to learn it's a perf problem (i.e. feedback mechanism), then we can leave it as is and improve if data suggests us so
- The most interesting data that I'd love not to lose is errors. So maybe we can start with a place where it's mostly performant (accumulates logs in memory), but flushes on any category="error" or end of run.